Avoid test_pretty_no_wrap_line failure on large screens

Flamefire opened the pull request:
#### Brief description of what is fixed or changed

20 expressions don't seem to be enough to ensure line wrapping happens reliably.

Test fails on an 2560x1440 screen with a full screen console but succeeds when shrinking the console

This disables the screen-width detection in the call done in the test such that the behavior is consistent


#### Release Notes

<!-- BEGIN RELEASE NOTES -->
NO ENTRY
<!-- END RELEASE NOTES -->

Flamefire committed:
Avoid test_pretty_no_wrap_line failure on large screens

20 expressions don't seem to be enough to ensure line wrapping happens reliably.

Test fails on an 2560x1440 screen with a full screen console but succeeds when shrinking the console

Flamefire committed:
Set number of columns instead of increasing expression length

Flamefire commented:
> It would be better to make it so that the test does not depend on the display size at all if possible.

Ok, I found the `num_cols` param. Setting it to 80 works too as this disables the auto-detection

oscarbenjamin commented:
You'll need to add a line like
```
Alexander Grund <alexander.grund@tu-dresden.de> Alexander Grund <Flamefire@users.noreply.github.com>
```
to the .mailmap file and then run `bin/mailmap_check.py` to sort it into otder.

Flamefire committed:
author: Add Alexander Grund to .mailmap

oscarbenjamin commented:
The authors check is failing with:
```
This author is not included in the .mailmap file:
Alexander Grund <Flamefire@users.noreply.github.com>
```
This is because this is the email address that is configured in GitHub:
https://docs.github.com/en/account-and-profile/how-tos/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address

You can change how it is configured in GitHub or add the line as I showed above.

Flamefire commented:
> This is because this is the email address that is configured in GitHub:

I actually wanted to avoid using that and updated the commit-author of the commit I made through the GitHub GUI. Locally the script does not complain anymore. But it seems the GitHub checkout action produces a temporary merge commit under my (GitHub) name for the CI run

Updated.
